src/polymer/polymer2-element-scanner.ts

```diff
@@ -380,7 +380,7 @@ class ClassFinder implements Visitor {
       return;
     }
 
-    const name = node.id && astValue.getIdentifierName(node.id);
+    const name = (node.id && astValue.getIdentifierName(node.id)) || undefined;
```

Comment:
`|| undefined` is unusual. Why is it necessary, and why for an update to Chai types?